src/location/mqttDeserialize.ts

```diff
@@ -1,6 +1,13 @@
 import * as t from 'io-ts';
 import queryStringParser from 'query-string';
 import { unsafeDecode } from '../common/typeUtil';
+import destination from '@turf/destination';
+
+const LIBRARY_BEARING = 42.5;
+const LIBRARY_ORIGO = {
+  lat: 60.205325,
+  lon: 24.962115,
+};
```

Comment:
I'd suggest a minor tweak to the coordinates of the origin

const LIBRARY_ORIGIN = {
  lat: 60.205318,
  lon: 24.962113
};

The difference is visible only when zooming real close, though.